### PR TITLE
docs(provider/oracle): add reference for Oracle Cloud

### PR DESCRIPTION
**Pull request by shihchang**
docs(provider/oracle): add reference for Oracle Cloud

### CODE REVIEW

File: reference/providers/oracle.md
Comment (dorbin) [inline reference/providers/oracle.md]: @shihchang 
Instead of linking to /tryit/, can you link to something more specifically about `user`? [This](https://docs.cloud.oracle.com/iaas/Content/GSG/Tasks/addingusers.htm#one) maybe?

File: reference/providers/oracle.md
Comment (dorbin) [inline reference/providers/oracle.md:48]: @shihchang , I'm just curious: what happens in the Spinnaker UI if you try to create a firewall?

**Review by dorbin — COMMENTED** [review]
LGTM, pending review by @lwander

File: reference/providers/oracle.md
Comment (lwander) [inline reference/providers/oracle.md]: Where does this bucket come from?

**Review by lwander — APPROVED** [review]

File: reference/providers/oracle.md
Comment (shihchang) [reply reference/providers/oracle.md:48]: It shows the messag, "We don't currently support the ability to create firewalls for Oracle Cloud from Spinnaker. You can use the Oracle Cloud console to create and manage Security Lists to provide a virtual firewall for an instance."

This is in the following PR
https://github.com/spinnaker/deck/pull/5868/files#diff-f8eb3d3a6ff49ce3d7f0f4084b69c8f2

File: reference/providers/oracle.md
Comment (shihchang) [reply reference/providers/oracle.md]: That link looks better, changed.

File: reference/providers/oracle.md
Comment (shihchang) [reply reference/providers/oracle.md]: Spinnaker clouddriver (Oracle provider) creates the "_spinnaker_server_group_data" bucket in Oracle Object Store within the user's tenancy.

Comment (lwander) [reply reference/providers/oracle.md]: I would document that

Comment (shihchang) [reply reference/providers/oracle.md]: Yes, we do have that, "The Oracle Cloud provider in Spinnaker creates a bucket named _spinnaker_server_group_data to store all the server group templates." Or do I misunderstand what you suggest to add to this section?

File: reference/providers/oracle.md
Comment (dorbin) [reply reference/providers/oracle.md:48]: :+1:

File: reference/providers/oracle.md
Comment (shihchang) [reply reference/providers/oracle.md]: I change this to "The Oracle Cloud provider in Spinnaker creates a bucket named _spinnaker_server_group_data within the user's tenancy to keep track of all the server groups created."  This is also to avoid using the unexplained term of "server group template"

File: reference/providers/oracle.md
Comment (dorbin) [inline reference/providers/oracle.md]: I notice you've removed mention of Oracle Object Storage. Are you assuming the reader (presumably an Oracle user) will know that's what you're referring to?

File: reference/providers/oracle.md
Comment (lwander) [reply reference/providers/oracle.md]: Ah sorry I must've misread this or the context. LGTM

**Review by lwander — APPROVED** [review]

File: reference/providers/oracle.md
Comment (shihchang) [reply reference/providers/oracle.md]: yes, I changed it to Oracle Object Storage bucket for clarity.

**Pull request merged**
